nifi-extension-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/drive/ListGoogleDrive.java

```diff
@@ -197,7 +266,37 @@ public void onScheduled(final ProcessContext context) throws IOException {
 
         HttpTransport httpTransport = new ProxyAwareTransportFactory(proxyConfiguration).create();
 
-        driveService = createDriveService(context, httpTransport, DriveScopes.DRIVE_METADATA_READONLY);
+        driveService = createDriveService(context, httpTransport, DriveScopes.DRIVE, DriveScopes.DRIVE_METADATA_READONLY);
+    }
+
+    @OnScheduled
+    @Override
+    // initListedEntityTracker() needs to be overridden because the super implementation calls getRecordSchema()
+    // so recordSchema must be initialized before that
```

Comment:
Method-level comments should follow Javadoc conventions, and be set before method annotations.

Reply:
Not an issue in the reimplemented version.